content/boilerplates/notes/1.1.9.md

```diff
@@ -2,7 +2,7 @@
 
 - Prevent overly large strings from being sent to Prometheus ([Issue 14642](https://github.com/istio/istio/issues/14642)).
 - Reuse previously cached JWT public keys if transport errors are encountered during renewal ([Issue 14638](https://github.com/istio/istio/issues/14638)).
-- Bypass JWT authentication for HTTP OPTIONS methods to support CORS requests ([Issue 2160](https://github.com/istio/proxy/issues/2160)).
+- Bypass JWT authentication for HTTP OPTIONS methods to support CORS requests (Issue 2160 (broken link: `https://github.com/istio/proxy/issues/2160`)).
```

Comment:
Just remove the parenthesized part. This link will never come back since we shut down the issue tracker in that  repo.